src/lib/error_json/error_json.ml

```diff
@@ -7,6 +7,23 @@ let rec sexp_to_yojson (sexp : Sexp.t) : Yojson.Safe.t =
   | List sexps ->
       `List (List.map ~f:sexp_to_yojson sexps)
 
+let sexp_record_to_yojson (sexp : Sexp.t) : Yojson.Safe.t =
```

Comment:
This function being unsafe to call scares me a bit, but should be fine here since the sexp should be stable. Almost seems better to just leave it as a sexp, though turning it into json does make it filterable, which might be nice. I'm trusting that this has already been tested locally to make sure this doesn't choke on a monitor sexp.